src/js/login/reducers/login.js

```diff
@@ -5,11 +5,10 @@ import {
   UPDATE_LOGIN_URL,
   UPDATE_VERIFY_URL,
   UPDATE_LOGOUT_URL,
-  UPDATE_SEARCH_HELP_USER_MENU
+  UPDATE_SEARCH_HELP_USER_MENU,
+  LOG_OUT
```

Comment:
I have the vague recollection that this needed to be shared across apps at some point but it looks like that's no longer the case.